package.json

```diff
@@ -28,22 +29,18 @@
   },
   "homepage": "https://github.com/migme/bubbly#readme",
   "devDependencies": {
-    "babel": "^5.6.23",
-    "babel-eslint": "^3.1.23",
-    "chai": "^3.2.0",
+    "babel-cli": "^6.6.5",
+    "babel-eslint": "^5.0.0",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "chai": "^3.5.0",
     "customevent-shim": "^0.1.0",
-    "event-target-shim": "^1.0.5",
-    "mocha": "^2.2.5",
-    "sinon": "^1.15.4",
-    "sinon-chai": "^2.8.0",
-    "standard": "^5.0.0-2"
-  },
-  "standard": {
-    "parser": "babel-eslint"
-  },
-  "browserify": {
-    "transform": [
-      "babelify"
-    ]
+    "eslint": "^2.4.0",
+    "eslint-config-migme": "^2.0.1",
```

Comment:
In my local I need to add ```"eslint-plugin-react": "^5.0.1"``` to make eslint run, ```eslint-config-migme``` seem to use ```'eslint-config-migme/rules/react'``` by default which need eslint-plugin-react